terraform/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "app_external_ip" {
+  value = google_compute_instance.app.network_interface.0.access_config.0.nat_ip
```

Comment:
Add the description of this output

https://cloud.google.com/docs/terraform/best-practices-for-terraform#outputs